requirements/requirements.in

```diff
@@ -29,7 +29,7 @@ Markdown==2.6.11    # used for formatting API help
 ordereddict==1.1
 prometheus_client==0.6.0
 psutil==5.4.3
-psycopg2==2.7.3.2   # problems with Segmentation faults / wheels on upgrade
+psycopg2==2.8.2
```

Comment:
nice comment

Reply:
For reference: https://github.com/psycopg/psycopg2/issues/543

Reply:
wow that issue has a whole lot of backlinks. It's closed, so do you have a sense of their resolution? At a quick look, it sounds like they recommended the package `psycopg2-binary` to do a wheel install. Could we switch to that and avoid the pin?

Reply:
https://pypi.org/project/psycopg2-binary/

ok, no

> The binary package is a practical choice for development and testing but in production it is advised to use the package built from sources.

I'm kind of unclear about where that leaves users of this library who want it in production.

Reply:
At ti's core, this is an issue between conflicting versions of libssl, and should manifest when you import psycopg2.  

Dev Container (CentOS 7): 
![image](https://user-images.githubusercontent.com/11698892/59513165-54263f80-8e88-11e9-9b8b-2cfd7a41305a.png)

I'll check to make sure this doesn't segfault on RHEL7 & RHEL8 too.

Reply:
oh yeah, I wasn't following before, but the upgrade here looks good 👍